parser/mft.go

```diff
@@ -437,7 +445,7 @@ func ParseMFTFile(
 				ParentEntryNumber:    file_names[0].MftReference(),
 				ParentSequenceNumber: file_names[0].Seq_num(),
 				FullPath:             full_path,
-				FileName:             longest_filename,
+				FileName:             display_filename,
```

Comment:
Can you just do
```
FileName: get_display_name(file_names),
```

and get rid of the above code?

Reply:
this shoudl be done in latest commit